src/libraries/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -1719,16 +1720,16 @@
     <value>Cannot load the schema from the location '{0}' - {1}</value>
   </data>
   <data name="Sch_LengthGtBaseLength" xml:space="preserve">
-    <value>It is an error if 'length' is among the members of {facets} of {base type definition} and {value} is greater than the {value} of the parent 'length'.</value>
+    <value>It is an error if 'length' is among the members of the facets of the base type definition and its value is greater than the value of the parent 'length'.</value>
```

Comment:
Could you word this and below message similarly,  for me this version sounds better

Reply:
@buyaa-n I've been trying to work on all this feedback, but I'm blocked by #34612 / Nuget/Home#7052

I'll get back to these as soon as I can get the test projects to build in Visual Studio again.

Reply:
@buyaa-n I think I've got a work-around.
I've reworded these two:
Sch_MinLengthGtBaseMinLength
Sch_MaxLengthGtBaseMaxLength
- will push when I have finished all requested changes.